app/views/help/cookies.html.haml

```diff
@@ -4,21 +4,111 @@
 
 %p.govuk-body= t('cookies.section_one.paragraph_two')
 
-%h2.govuk-heading-l= t('cookies.section_two.section_heading')
+%h2.govuk-heading-l= t('cookies.section_necessary.section_heading')
 
-%p.govuk-body= t('cookies.section_two.paragraph_one')
+%p.govuk-body= t('cookies.section_necessary.paragraph_one')
 
-%p.govuk-body= t('cookies.section_two.paragraph_two')
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = t('cookies.section_necessary.table.caption')
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{:scope => "col"}
```

Comment:
Standard: use the new json style hash syntax instead of the rocket hash syntax

Reply:
Done 👍